src/demos/calcite-flow-item.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
@asangma these demo files should be removed. Only demos within their component folder should exist.

Reply:
Done.